### DISCUSSION TITLE
Default admin account name

### DISCUSSION THREAD

**Discussion by pkaraalphalayer** [LizardByte/community#134] [Sunshine] [answered]
When I go to http://localhost:47990, it asks for a username/password.  I don't recall what the default username is?  Anyone know?

And say I needed to reset this, would just unstalling/reinstalling allow me to reconfigure?  or do I need to delete some registry element

**Comment by ReenigneArcher** [comment, marked as answer]
`sunshine` is the default username... there's not a default password

https://docs.lizardbyte.dev/projects/sunshine/en/latest/troubleshooting/general.html#forgotten-credentials

**Reply by pkaraalphalayer** [reply]
thank you!
